templates/site/node.html

```diff
@@ -3,74 +3,64 @@
+            <div class="row">
+                <div class="col-9">
+                    <!--H2 title -->
+                    <div id="name" class="mb-0"></div>
+                    <!--H3 title-->
+                    <div id="affiliation" class="mb-1 text-body-secondary"></div>
+                    <div id="description"></div>
+                    <div class="mt-3">
+                        <table>
+                            <thead>
+                            <tr>
+                                <th></th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            <tr>
+                                <td>
+                                    <div id="url"></div>
+                                </td>
+                            </tr>
+                            <tr>
+                                <td>Date Added: <span id="last_updated"></span></td>
+                            </tr>
+                            <tr>
+                                <td>Status: <span id="status"></span></td>
+                            </tr>
+                            <tr>
+                                <td>Contact: <span id="contact"></span></td>
+                            </tr>
+                            <tr>
+                                <td>Version: <span id="version"></span></td>
+                            </tr>
+                            </tbody>
+                        </table>
                     </div>
                 </div>
-                <div class="row">
-                    <div class="col-12 text-center">
-                        <h4 class="text-decoration-underline">Services Available</h4>
-                    </div>
+                <div class="col-3 col-lg-2 d-block d-lg-block">
+                    <img id="image-right" class="w-sm-100 mt-2 px-auto d-block img-fluid {{ img_class|default('') }}" src="{{ img_src }}" />
                 </div>
-                <div class="row">
-                    <div class="col-12">
-                        <div id="services" class="table-responsive"></div>
-                    </div>
+            </div>
+            <div class="row">
+                <div class="col-12 mt-2 text-center">
+                    <h4 class="text-decoration-underline">Services Available</h4>
                 </div>
             </div>
-            <div class="col-1 col-sm-4 d-none d-lg-block overflow-hidden shadow-lg">
-                <!--Sets logo as background image-->
-                <div id="background-icon">
-                    <!--Div used to apply blur effect to background image -->
-                    <div id="node-overlay-container" class="foreground-img-blur mb-auto">
-                        <!--Used to set sharp logo overlayed on blurred logo.  May be used in future -->
-                        <!--<img id="node-overlay-img" > -->
-                    </div>
+            <div class="row">
```

Comment:
why do we need an additional row? why not just round the last row that has content?

Reply:
There should only be 12 columns in a row.  Any more and a new row should be used.

Reply:
nevermind. Ignore this comment. I was confused by the padding